web/src/routes/System.jsx

```diff
@@ -347,23 +349,26 @@ export default function System() {
 
           <Heading size="lg">Other Processes</Heading>
           <div data-testid="cameras" className="grid grid-cols-1 3xl:grid-cols-3 md:grid-cols-2 gap-4">
-            {['go2rtc', 'recording'].map((process) => (
+            {processesNames.map((process) => (
               <div key={process} className="dark:bg-gray-800 shadow-md hover:shadow-lg rounded-lg transition-shadow">
                 <div className="capitalize text-lg flex justify-between p-4">
                   <div className="text-lg flex justify-between">{process}</div>
+                  <p>P-ID: {processes[process]['pid'] || '- '}</p>
```

Comment:
This should be done the same way as other cards as part of the table

Reply:
If you reference to other cards, all "other processes" should be a table inside one card in this logic...

Reply:
I don't think so. For example when you have multiple detectors 

![Screenshot_20230502_205418_Chrome Beta.jpg](https://user-images.githubusercontent.com/14866235/235824645-e9f1c4f6-c6e4-4d7b-a570-696067fb7ea2.jpg)

Reply:
make sense... (https://github.com/blakeblackshear/frigate/pull/6344/commits/4adf482475823ce583f77218401ab883d8a40653)
<img width="2032" alt="image" src="https://user-images.githubusercontent.com/15078499/235826035-474ecd03-14bb-41ec-8d03-a752db024381.png">